cmd/helm/template.go

```diff
@@ -77,10 +81,50 @@ func newTemplateCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 			if rel != nil {
 				var manifests bytes.Buffer
 				fmt.Fprintln(&manifests, strings.TrimSpace(rel.Manifest))
-
-				if !client.DisableHooks {
-					for _, m := range rel.Hooks {
-						fmt.Fprintf(&manifests, "---\n# Source: %s\n%s\n", m.Path, m.Manifest)
+				newDir := client.OutputDir
```

Comment:
This section should probably be put in a new function or several new functions. The cyclomatic complexity of newTemplateCmd is already too high, and this makes it _really_ hard to read.

Reply:
All right! I will refactor this function to reduce its complexity .